src/components/FAQ/index.jsx

```diff
@@ -0,0 +1,148 @@
+import { Box, makeStyles, Typography } from '@material-ui/core'
+import React, { useState } from 'react'
+import Accordion from '@material-ui/core/Accordion'
+import AccordionDetails from '@material-ui/core/AccordionDetails'
+import AccordionSummary from '@material-ui/core/AccordionSummary'
+import AddCircleIcon from '@material-ui/icons/AddCircle'
+import RemoveCircleIcon from '@material-ui/icons/RemoveCircle'
+import DUMMY_QA from '../../data/FAQs,js'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    minHeight: '100vh',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    backgroundColor: theme.palette.background.secondary,
+    padding: 16,
+  },
+  title: {
+    color: theme.palette.tertiary.dark,
+    marginTop: '4rem',
+    textAlign: 'center',
+  },
+  subtitle: {
+    textAlign: 'center',
+  },
+  section: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '60vw',
+    padding: '1rem',
+
+    '& .MuiAccordion-root': {
+      marginTop: '.8rem',
+      borderRadius: '10px',
+    },
+    '& .MuiAccordion-root:before': {
+      position: 'inherit',
+    },
+    '& .MuiAccordion-root.Mui-expanded': {
+      margin: '1rem 0 0 0',
+    },
+    '& .MuiAccordionSummary-content': {
+      paddng: '0 0.8rem 0 0.8rem',
+      margin: 0,
+    },
+    '& .MuiAccordionDetails-root': {
+      padding: '1rem 1rem 0 1rem',
```

Comment:
```suggestion
      padding: '1rem 1.8rem 0 1.8rem',
```

I guess can adjust to the question alignment, seems more neat with more padding around
![Screenshot 2022-01-29 at 9 46 58 PM](https://user-images.githubusercontent.com/77470406/151663501-6dc8dbde-f992-4657-a25c-85ef0ff680aa.png)

Reply:
Okay!